webapp/src/DataTransferObject/AccessEndpoint.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace App\DataTransferObject;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class AccessEndpoint
+{
+    /**
+     * @param string[] $properties
+     */
+    public function __construct(
+        public readonly string $type,
+        #[Serializer\Type('array<string>')]
```

Comment:
```suggestion
        #[Serializer\Type('string[]')]
```

Reply:
Not sure if this is valid, if it is we have more places where this should/can be done.

Reply:
Nope doesn't work.